GitUI/WindowsJumpListManager.cs

```diff
@@ -29,6 +29,10 @@ public sealed class WindowsJumpListManager : IDisposable
         public WindowsJumpListManager(IRepositoryDescriptionProvider repositoryDescriptionProvider)
         {
             _repositoryDescriptionProvider = repositoryDescriptionProvider;
+            if (TaskbarManager.IsPlatformSupported)
+            {
+                TaskbarManager.Instance.ApplicationId = "GitExtensions";
```

Comment:
Do we have a constant for this?

Reply:
No, can add one. Where?

Reply:
> I vaguely remember we have a const for "GitExtensions".
Not a big deal

This must match the id that associates to .gitext in the installer and probably should match to the installer ProgId, various AssemblyProduct and hardcoded paths in AppSettings.
But AppSettings is the most sane, I will add it there for now.